### PR TITLE
DRY codec.rs

### PR DESCRIPTION
**Pull request by zserik**
Reduce repetition in `codec.rs` + correct formatting using `cargo fmt`.

### CODE REVIEW

**Comment by zserik** [comment]
[clabot:check]

**Comment by twittner** [comment]
Sorry for the conflict, but #18 was already good to go in. If you want feel free to resubmit your PR on top of the current master, but please, do *not* apply rustfmt.

**Comment by twittner** [comment]
I would really appreciate it if your PR would not mix formatting with other changes. It blows up the diff for no good reason. So again, please do not apply rustfmt.

File: src/codec.rs [hunk 1]
```diff
@@ -132,28 +130,27 @@ impl<T> UviBytes<T> {
 
     fn deserialise(&mut self, src: &mut BytesMut) -> Result<Option<BytesMut>, io::Error> {
         loop {
-            match self.len.take() {
+            return match self.len.take() {
```
Comment (twittner) [inline src/codec.rs]: I would prefer not to make the proposed changes to `deserialise`. While functionally equivalent, stylistically they do not read as an improvement to me.

Comment (zserik) [reply src/codec.rs]: ok, done.

**Comment by twittner** [comment]
Thank you!

**Pull request merged**